### PR TITLE
WebAssembly roundup blog post series

### PR DESCRIPTION
**Pull request by georgestagg**

### CODE REVIEW

**Review by hadley — APPROVED** [review]
These look great overall! I only saw a few places for minor improvements.

File: content/blog/quarto-live-0-1-1/index.Rmd [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+* [x] Create `thumbnail-sq.jpg`; height and width should be equal
+* [x] Create `thumbnail-wd.jpg`; width should be >5x height
+* [x] `hugodown::use_tidy_thumbnails()`
+* [x] Add intro sentence, e.g. the standard tagline for the package
+* [x] `usethis::use_tidy_thanks()`
+-->
+
+We're tickled pink to announce the release of [Quarto Live](https://r-wasm.github.io/quarto-live/) 0.1.1. Quarto Live is a new Quarto extension that uses WebAssembly to bring interactive examples and code exercises with custom grading algorithms to your HTML-based output documents, using standard Quarto markdown syntax.
+
+Quarto Live adds a [CodeMirror](https://codemirror.net/)-based text editor to your document with automatic theming, syntax highlighting, and auto-complete. The editor executes R code using webR, and even integrates with Quarto's OJS support so that interactive code cells update reactively with other `ojs` cells running in the page.
```
[the remark this answers is not in the document]
Comment (georgestagg) [reply content/blog/quarto-live-0-1-1/index.Rmd]: Good idea, I've linked to https://quarto.org/docs/interactive/ojs/.

File: content/blog/quarto-live-0-1-1/index.Rmd [hunk 2]
```diff
@@ -0,0 +1,260 @@
...
+```
+
+
+The resulting rendered document looks like the screenshot below. An editor is inserted into the document in the place of the code block, and an interested reader can use it to modify and re-execute the source code in place.
+
+<img src="images/ggplot-1.png" alt="Screenshot showing the result of rendering the example document above. The Quarto Live editor is shown pre-populated with the provided code snippet. An output graphic is shown, showing the result of plotting the airquality dataset using the ggplot2 package."/>
+
+Quarto Live executes R code using the [evaluate](https://evaluate.r-lib.org) package, with output rendered using functions from [knitr](https://yihui.org/knitr/), so the output should be almost identical to output generated by R Markdown or Quarto.
+
+The real beauty of this, in my opinion, is that the reader does not have to install any packages, copy and paste source code, switch to an IDE like RStudio, or deal with myriad other small but fiddly distractions just to experiment with a new piece of code or R package. They can do it, right there, without any context switching.
```
[the remark this answers is not in the document]
Comment (georgestagg) [reply content/blog/quarto-live-0-1-1/index.Rmd]: I've linked to https://positron.posit.co for Positron. Then RStudio looked a little strange unlinked so have also linked it to https://posit.co/products/open-source/rstudio/.

File: content/blog/quarto-live-0-1-1/index.Rmd [hunk 3]
```diff
@@ -0,0 +1,260 @@
...
+You can add the Quarto Live extension to a project by running the following command in a terminal with the current directory of a Quarto project:
+
+```
+quarto add r-wasm/quarto-live
+```
+
+Then, create a new document with the following template to get up and running using the `knitr` engine:
+
+<!--
+  This rather strange way of displaying output throughout this post is to
```
[the remark this answers is not in the document]
Comment (georgestagg) [reply content/blog/quarto-live-0-1-1/index.Rmd:53]: I tried with:

`````
````
```{webr}
library(ggplot2)
[...]
```
````
`````

but knitr still tries to kick in inside the ````` ```` ````` block :(

IIRC in Quarto you can do:
`````
````
```{{webr}}
library(ggplot2)
[...]
```
````
`````

Then the double braces are escaped and displayed as single braces without knitr kicking in, but in Rmarkdown the double braces are retained in the output.

**Pull request merged**